src/rdkafka_admin.c

```diff
@@ -2585,6 +2634,26 @@ rd_kafka_ConfigResource_delete_config(rd_kafka_ConfigResource_t *config,
                                           RD_KAFKA_ALTER_OP_DELETE);
 }
 
+rd_kafka_resp_err_t
+rd_kafka_ConfigResource_incremental_remove_config(rd_kafka_ConfigResource_t *config,
+                                   const char *name) {
+        if (!name || !*name)
+                return RD_KAFKA_RESP_ERR__INVALID_ARG;
+
+        return rd_kafka_admin_incremental_add_config0(&config->config, name, NULL,
+                                          RD_KAFKA_INCREMENTAL_ALTER_OP_REMOVE);
+}
+
+rd_kafka_resp_err_t
+rd_kafka_ConfigResource_incremental_subtract_config(rd_kafka_ConfigResource_t *config,
+                                   const char *name) {
+        if (!name || !*name)
+                return RD_KAFKA_RESP_ERR__INVALID_ARG;
+
+        return rd_kafka_admin_incremental_add_config0(&config->config, name, NULL,
+                                          RD_KAFKA_INCREMENTAL_ALTER_OP_SUBTRACT);
+}
```

Comment:
incremental subtract does expect a value string as well just like set and append. So basically it omits only the mentioned values in the list. 
Eg: key1->[v1,v2,v3] 
Subtract(key1,"v1,v3")
Key1->[v2]